check/check.go

```diff
@@ -88,38 +96,50 @@ func (c *Check) Run() {
 	cs := c.Commands
 
 	// Initialize command pipeline
-	cs[0].Stderr = os.Stderr
 	cs[n-1].Stdout = &out
 	i := 1
 
 	var err error
+	errmsgs = ""
+
 	for i < n {
 		cs[i-1].Stdout, err = cs[i].StdinPipe()
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "%s: %s\n", cs[i].Args, err)
-		}
+		errmsgs += handleError(
+			err,
+			fmt.Sprintf("check.Run: Audit %s failed\nfailing command: %s",
+				c.Audit,
+				cs[i].Args,
+			),
+		)
 
-		cs[i].Stderr = os.Stderr
 		i++
 	}
 
 	// Start command pipeline
 	i = 0
 	for i < n {
 		err := cs[i].Start()
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "%s: %s\n", cs[i].Args, err)
-		}
+		errmsgs += handleError(
+			err,
+			fmt.Sprintf("check.Run: Audit %s failed\nfailing command: %s",
```

Comment:
I try to capture 2 pieces of info with that error message:
1. The entire audit pipeline that failed.
2. Which specific command in that pipeline failed.

So something like:
`
Failed to run: commandline
Failing command: actual command
`